src/main/java/me/rigamortis/seppuku/impl/module/render/NametagsModule.java

```diff
@@ -0,0 +1,456 @@
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.item.*;
+import net.minecraft.entity.monster.IMob;
+import net.minecraft.entity.passive.IAnimals;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraft.potion.Potion;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.StringUtils;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.Vec3d;
+import team.stiff.pomelo.impl.annotated.handler.annotation.Listener;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Author Seth
+ * 4/20/2019 @ 10:07 AM.
+ */
+public final class NametagsModule extends Module {
+
+    public final BooleanValue players = new BooleanValue("Players", new String[]{"Player"}, true);
+    public final BooleanValue mobs = new BooleanValue("Mobs", new String[]{"Mob"}, true);
+    public final BooleanValue animals = new BooleanValue("Animals", new String[]{"Animal"}, true);
+    public final BooleanValue vehicles = new BooleanValue("Vehicles", new String[]{"Vehic", "Vehicle"}, true);
+    public final BooleanValue local = new BooleanValue("Local", new String[]{"Self"}, true);
+    public final BooleanValue items = new BooleanValue("Items", new String[]{"Item"}, true);
+    public final BooleanValue armorStand = new BooleanValue("ArmorStands", new String[]{"ArmorStand", "ArmourStand", "ArmourStands", "ArmStand"}, true);
+
+    public final BooleanValue name = new BooleanValue("Name", new String[]{"Nam"}, true);
+    public final BooleanValue ping = new BooleanValue("Ping", new String[]{"Ms"}, true);
+    public final BooleanValue armor = new BooleanValue("Armor", new String[]{"Arm"}, true);
+    public final BooleanValue health = new BooleanValue("Health", new String[]{"Hrts"}, true);
+    public final BooleanValue enchants = new BooleanValue("Enchants", new String[]{"Ench"}, true);
+    public final OptionalValue potions = new OptionalValue("Potions", new String[]{"Pot", "Pots", "PotsMode"}, 1, new String[]{"None", "Icons", "Text"});
+
+    public final BooleanValue background = new BooleanValue("Background", new String[]{"Bg"}, true);
+
+    private ICamera camera = new Frustum();
+    private final ResourceLocation inventory = new ResourceLocation("textures/gui/container/inventory.png");
+
+    //i cba
+
+    public NametagsModule() {
+        super("Nametags", new String[]{"Tags", "Name"}, "Nametags", "NONE", -1, ModuleType.RENDER);
+    }
+
+    @Listener
+    public void render2D(EventRender2D event) {
+        final Minecraft mc = Minecraft.getMinecraft();
+
+        List<Entity> entitylist = WallHackModule.entitylist;
```

Comment:
What happens if someone never turns on WallHack?  Your list will always be empty until they do, which isn't obvious to the user.

Reply:
Also, please follow camel-case naming conventions when submitting code changes to Seppuku.